pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -1674,6 +1674,22 @@ let
 
       ms-vscode.cpptools = callPackage ./cpptools { };
 
+      ms-vscode.PowerShell = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          name = "PowerShell";
+          publisher = "ms-vscode";
+          version = "2022.6.3";
+          sha256 = "sha256-P8kakmpT3yAkKFroKttiLOV7PneOIoQOEtwImPIDHQY=";
+        };
+        meta = {
+          description = "PowerShell Language Support for Visual Studio Code.";
```

Comment:
```suggestion
          description = "PowerShell Language Support for Visual Studio Code";
```

Reply:
Please remove "." from description to comply with guidelines:

https://nixos.org/manual/nixpkgs/stable/#var-meta-description

Reply:
Description could be improved too:
`A Visual Studio Code extension for PowerShell language support` (feel free to improve it further)